rules/code-quality/src/Rector/Ternary/ArrayKeyExistsTernaryThenValueToCoalescingRector.php

```diff
@@ -76,6 +77,10 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
+        if ($node->else instanceof Variable) {
+            return null;
+        }
```

Comment:
I think that if `$node->else` is anything but null, it should be skipped.

Could you add also test fixture where else is string or false?

Reply:
I will try.

Reply:
Ah. Seeing this, the else can be anything and it's correct.
I got confused, sorry.


What exactly is the bug here?

Reply:
I think when else is a variable, we should skip that, not sure about false or string or other value..., /cc @infabo for verification

Reply:
I've added handle else is not null to be skipped.

Reply:
The following stil not checked: 

```php
return array_key_exists('value', $v) ? $v['value'] : 'null';
```

I will re-check.

Reply:
I've added handling for string "null"